app/styles/nypr-member-center.scss

```diff
@@ -13,6 +13,8 @@
 @import "nypr-member-center/current-status-detail";
 @import "nypr-member-center/loading-animation";
 
+@import "nypr-account-modal";
+
```

Comment:
no, I think account settings is its own self contained package that shouldn't have any other top level exports besides `nypr-account-settings`.

@mike-hearn has a PR that's open to address this, but given the pending nature of account settings, let's let this merge in and we can rebase #17 to include the new work you've added.